includes/documents/class-wcpdf-packing-slip.php

```diff
@@ -159,14 +159,16 @@ public function init_settings() {
 	 * Document number title
 	 */
 	public function get_number_title() {
-		return apply_filters( "wpo_wcpdf_{$this->slug}_number_title", __( 'Packing Slip Number', 'woocommerce-pdf-invoices-packing-slips' ), $this );
+		$number_title = trim( __( 'Packing Slip Number:', 'woocommerce-pdf-invoices-packing-slips' ), ':' );
```

Comment:
We don't have this string present in the free plugin (since the functionality is not included), same for the date string.
We could keep this here, but then we should decide what to do with the translations that we currently have in the `wpo_wcpdf_pro` domain.

Reply:
What you prefer? Have everything equal for consistence?

Reply:
Yes, for the base plugin that's probably a good idea -> that way we have all the documents conform to a certain standard. I just wanted to mention it so that we keep it in mind for the "pro" end of things (maybe open a separate issue there).

Reply:
But you want to add the colon also to Pro?

Reply:
This comment wasn't about the colon but about the fact that you have now inserted this here in the `woocommerce-pdf-invoices-packing-slips` domain but it wasn't part of that domain yet (so translations do not yet exist). The string itself is only used in pro, where it is currently in the `wpo_wcpdf_pro` domain (with a colon).
I personally like the idea of having the string with a colon included pretranslated.

Reply:
Done here: https://github.com/wpovernight/woocommerce-pdf-ips-pro/commit/d9f766c54827c33696436592f1ca4d31de71e174